test2.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import fnmatch
+import os
+import random
+import string 
+
+folder1 = 'rrrrrr'
+folder2 = 'android'
+folder3 = 'v4'
+folder4 = 'immersion'
+folder5 = 'InPlay'
+folder6 = 'a'
+folder7 = 'bb'
+folder8 = 'cf'
+class TestProject(unittest.TestCase):
+	def setUp(self):
+		pass
+		
+	def test_foldername_rrrrrr(self):
+		for i in range(0,len(folder1)):
+			self.assertTrue(i+1<=(len(folder1)-1) and folder1[i] == folder1[i+1] and folder1.islower())

```

Comment:
The assertion condition is always the same for all the tests: it can be put in a helper method to increase readability